src/windowmanagement/HelpMenu.tsx

```diff
@@ -1,26 +1,144 @@
-import { Button, Menu } from '@mantine/core';
+import { Anchor, Button, Grid, Image, Menu, Modal, Text, Title } from '@mantine/core';
+import { useDisclosure } from '@mantine/hooks';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+import {
+  BookIcon,
+  ConsoleIcon,
+  ExitAppIcon,
+  FeedbackIcon,
+  InformationCircleOutlineIcon,
+  OpenInBrowserIcon,
+  RouteIcon
+} from '@/icons/icons';
+import { useAppSelector } from '@/redux/hooks';
 
 export function HelpMenu() {
+  const [opened, { open, close }] = useDisclosure(false);
+  const luaApi = useOpenSpaceApi();
+  const openSpaceVersion = useAppSelector((state) => state.version.openSpaceVersion);
+
+  async function toggleLuaConsole() {
+    if (!luaApi) {
+      return;
+    }
+    const visible = (await luaApi.propertyValue('LuaConsole.IsVisible')) as
+      | boolean
+      | undefined;
+    luaApi.setPropertyValueSingle('LuaConsole.IsVisible', !visible);
+  }
+
+  async function openGuiInBrowser() {
+    if (!luaApi) {
+      return;
+    }
+    const portProperty = (await luaApi.propertyValue('Modules.WebGui.Port')) as
+      | number
+      | undefined;
+    const port = portProperty ?? 4670; // TODO: Change this to correct port when hosted
+    const addressProperty = (await luaApi.propertyValue('Modules.WebGui.Address')) as
+      | string
+      | undefined;
+    const address = addressProperty ?? 'localhost';
+
+    const link = `http://${address}:${port}`;
+    window.open(link, '_blank');
+  }
+
+  function toggleShutdown() {
+    luaApi?.toggleShutdown();
+  }
+
+  function osVersionNumber(): React.JSX.Element {
+    if (!openSpaceVersion) {
+      return <Text>Fetching OpenSpace version...</Text>;
+    }
+
+    function formatVersion(version: {
+      major: number;
+      minor: number;
+      patch: number;
+    }): string {
+      return version.major !== 255 && version.minor !== 255 && version.patch !== 255
+        ? `${version.major}.${version.minor}.${version.patch}`
+        : 'Custom';
+    }
+
+    return <Text>OpenSpave version: {formatVersion(openSpaceVersion)}</Text>;
+  }
+
   return (
-    <Menu position={'bottom-start'} offset={4} withArrow arrowPosition={'center'}>
-      <Menu.Target>
-        <Button size={'xs'} color={'gray'}>
-          Help
-        </Button>
-      </Menu.Target>
-      <Menu.Dropdown>
-        <Menu.Item>Open Web Tutorials</Menu.Item>
-        <Menu.Item>Open Getting Started Tour</Menu.Item>
-        <Menu.Divider />
-        <Menu.Item>Send Feedback</Menu.Item>
-        <Menu.Divider />
-        <Menu.Item>Toggle Console</Menu.Item>
-        <Menu.Item>Open GUI in Browser</Menu.Item>
-        <Menu.Divider />
-        <Menu.Item>Quit OpenSpace</Menu.Item>
-        <Menu.Divider />
-        <Menu.Item>About</Menu.Item>
-      </Menu.Dropdown>
-    </Menu>
+    <>
+      <Modal opened={opened} onClose={close} title={'About'} size={'40%'}>
```

Comment:
Make the title About OpenSpace instead? The title looks very small compared to the header